public/app/features/variables/inspect/utils.test.ts

```diff
@@ -0,0 +1,47 @@
+import { getPropsWithVariable } from './utils';
+
+describe('monkey', () => {
```

Comment:
nit: not sure this is the best name for the test 🐒

Reply:
hahaha, maybe I'll change it to `salmon` instead :) (ping @Korla)

Reply:
Hahaha, most definitely change before merging. As is, it is not the best name for the test @hugohaggmark.